frontend-build-pipeline.yml

```diff
@@ -16,30 +29,30 @@ steps:
 
 - script: |
     npm ci
-    npx ng build --prod
+    npx ng build
```

Comment:
let's do this in two separate steps, because each one takes a moderate amount of time and could fail for its own reasons, and because if somehow npm ci failed but ng build succeeded, we'd want to fail the build anyway since that would be weird.

Reply:
Alright, separated into different steps.